pkg/crc/oc/oc.go

```diff
@@ -0,0 +1,48 @@
+package oc
+
+import (
+	"fmt"
+	"path/filepath"
+	"strings"
+
+	"github.com/code-ready/crc/pkg/crc/constants"
+	crcos "github.com/code-ready/crc/pkg/os"
+)
+
+type OC struct {
+	OcBinaryPath   string
+	KubeconfigPath string
+}
+
+func GetOC(machineName string) OC {
```

Comment:
so, the function name is too ambigious. it should be something like `UseOCWithConfig`, right?